src/Hosting/Server.php

```diff
@@ -219,4 +219,41 @@ public function setServices($services)
         }
         $this->services = $services;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isDatabaseServer()
+    {
+        $services = $this->getServices();
+        return isset($services['database']);
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * Cloud API result is not explicit so try to guess based on the server name.
+     */
+    public function isFileServer()
+    {
+        return (bool) preg_match('/^fs-/', $this->getName());
```

Comment:
We can't assume fsdb are fileservers, for the same reason that we can't assume a ded is a webserver.